h2o-docs/src/product/data-science/algo-params/monotone_constraints.rst

```diff
@@ -1,7 +1,7 @@
 ``monotone_constraints``
 ------------------------
 
-- Available in: GBM, XGBoost
+- Available in: AutoML, GBM, XGBoost
 - Hyperparameter: no
 
 Description
```

Comment:
This topic currently has a note: 
**Note**: This option can only be used when the distribution is either ``gaussian`` or ``bernoulli``.

Update that note to:
**Note**: In GBM and XGBoost, this option can only be used when the distribution is either ``gaussian`` or ``bernoulli``.